docs/content/tutorials/index.md

```diff
@@ -186,25 +193,13 @@ The sample data has the following columns, and an example event is shown below:
 }
 ```
 
-The following tutorials demonstrate various methods of loading data into Druid, including both batch and streaming use
-cases. All tutorials assume that you are using the `micro-quickstart` single-machine configuration mentioned above.
-
-### [Tutorial: Loading a file](./tutorial-batch.html)
-
-This tutorial demonstrates how to perform a batch file load, using Druid's native batch ingestion.
-
-### [Tutorial: Loading stream data from Apache Kafka](./tutorial-kafka.html)
-
-This tutorial demonstrates how to load streaming data from a Kafka topic.
-
-### [Tutorial: Loading a file using Apache Hadoop](./tutorial-batch-hadoop.html)
-
-This tutorial demonstrates how to perform a batch file load, using a remote Hadoop cluster.
-
-### [Tutorial: Loading data using Tranquility](./tutorial-tranquility.html)
-
-This tutorial demonstrates how to load streaming data by pushing events to Druid using the Tranquility service.
+### Tutorials
```

Comment:
does it make sense for the title to indicate that these are loading data related?